tasks/aliases.json

```diff
@@ -0,0 +1,7 @@
+{
+  "default": ["build", "lint", "test", "watch"],
```

Comment:
Nope, points to the `watch` multitask in [watch.js](https://github.com/DFurnes/neue/blob/efe41fb0bcb4dfd10f0718c187fa20120e107e6a/tasks/watch.js).